apps/vs-code-designer/src/app/commands/createNewCodeProject/createCodeProjectSteps/createLogicApp/ScriptProjectCreateStep.ts

```diff
@@ -0,0 +1,120 @@
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { gitignoreFileName, hostFileName, localSettingsFileName } from '../../../../../constants';
+import { addDefaultBundle } from '../../../../utils/bundleFeed';
+import { confirmOverwriteFile, writeFormattedJson } from '../../../../utils/fs';
+import { ProjectCodeCreateStepBase } from '../../CodeProjectBase/ProjectCodeCreateStepBase';
+import { nonNullProp } from '@microsoft/vscode-azext-utils';
+import type { IActionContext } from '@microsoft/vscode-azext-utils';
+import type { IHostJsonV1, IHostJsonV2, ILocalSettingsJson, IProjectWizardContext } from '@microsoft/vscode-extension';
+import { FuncVersion } from '@microsoft/vscode-extension';
+import * as fse from 'fs-extra';
+import * as os from 'os';
+import * as path from 'path';
+import type { Progress } from 'vscode';
+
+/**
+ * This class represents a step that creates the contents of a new script project.
+ */
+export class ScriptProjectCreateStep extends ProjectCodeCreateStepBase {
+  // Set of files that should be ignored when creating a script project.
+  protected funcignore: string[] = [
+    '__blobstorage__',
+    '__queuestorage__',
+    '__azurite_db*__.json',
+    '.git*',
+    '.vscode',
+    'local.settings.json',
+    'test',
+    '.debug',
+  ];
+  protected gitignore = '';
+  protected supportsManagedDependencies = false;
+
+  /**
+   * Executes the step to create the contents of the project.
+   * @param context The project wizard context.
+   * @param _progress The progress object to use for reporting progress.
+   */
+  public async executeCore(
+    context: IProjectWizardContext,
+    _progress: Progress<{ message?: string | undefined; increment?: number | undefined }>
+  ): Promise<void> {
+    // Get the version of the Azure Functions runtime
+    const version: FuncVersion = nonNullProp(context, 'version');
+
+    // Create the host.json file
+    const hostJsonPath: string = path.join(context.projectPath, hostFileName);
+    if (await confirmOverwriteFile(context, hostJsonPath)) {
+      const hostJson: IHostJsonV2 | IHostJsonV1 = version === FuncVersion.v1 ? {} : await this.getHostContent(context);
+      await writeFormattedJson(hostJsonPath, hostJson);
+    }
+
+    // Create the local.settings.json file
+    const localSettingsJsonPath: string = path.join(context.projectPath, localSettingsFileName);
+    const logicAppFolderPath = context.logicAppFolderPath;
+    if (await confirmOverwriteFile(context, localSettingsJsonPath)) {
+      const localSettingsJson: ILocalSettingsJson = {
+        IsEncrypted: false,
+        Values: {
+          AzureWebJobsStorage: 'UseDevelopmentStorage=true',
+          FUNCTIONS_WORKER_RUNTIME: 'dotnet-isolated',
```

Comment:
Let's follow that src/commands/createNewProject/ProjectCreateStep/ScriptProjectCreateStep.ts line 44 follows on the functions worker runtime